pkg/util/testing/units.go

```diff
@@ -21,3 +21,10 @@ const (
 	Mi = Ki * Ki
 	Gi = Ki * Mi
 )
+
+func MessageOrEmpty(err error) string {
```

Comment:
even if it's too much effort to get rid of string comparisons, I prefer we don't put this in the utils, otherwise people will keep using it instead of doing proper error comparisons.

Reply:
That makes sense.
I will remove it from utils.

Reply:
done.